az-digital/az_quickstart
# #133 Event Content Type

This PR adds an event content type. `smart_date` is used as the contrib module for handling recurring dates.

## Description

- [x] Selection of range-recurrence date module, (eg `smart_date`, `date_recur`, etc)
- [x] Compatibility of Date AP Style with selected range-recurrence module
- [x] Row display mode
- [x] User permissions for events
- [x] Additional themeing
- [x] Event image styles
- [x] Basic event view

## Related Issue
#133 
#351 
#303 
#302 

## How Has This Been Tested?

- Create event
- Upload media for event and create a Large crop to test cropping
- Verify full content is rendered appropriately
- Verify that `/calendar` view contains the content


## Types of changes


## Checklist:
